common/defs/bitcoin/crown.json

```diff
@@ -33,9 +33,9 @@
   "min_address_length": 36,
   "max_address_length": 40,
   "bitcore": [
-    "https://insight-01.crown.tech",
-    "https://insight-02.crown.tech",
-    "https://insight-03.crown.tech"
+    "https://insight-01.crownplatform.com",
```

Comment:
These new bitcore addresses redirect back to crown.tech. Fix that or change these URLs back to crown.tech